### ISSUE TITLE
Using FASTLANE_EXPLICIT_OPEN_SIMULATOR with scan

### ISSUE THREAD

**Issue by 0xced** [fastlane/fastlane#6454]
### New Issue Checklist
- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
### Issue Description

By quickly reading the source code of scan, it looks like setting the `FASTLANE_EXPLICIT_OPEN_SIMULATOR` environment variable should at least print `Killing all running simulators` when running scan.

When running the following command, I don’t see anything related to killing and launching simulators.

```
$ FASTLANE_EXPLICIT_OPEN_SIMULATOR=1 SCAN_OUTPUT_TYPES='' scan --verbose --destination 'platform=iOS Simulator,name=iPhone 5s'
```

<details>
<summary>

scan output</summary>

```
INFO [2016-10-07 15:52:30.51]: $ xcodebuild -list -project XCDYouTubeKit.xcodeproj
INFO [2016-10-07 15:52:31.41]: $ xcodebuild clean -showBuildSettings -scheme XCDYouTubeKit\ Framework -project XCDYouTubeKit.xcodeproj
WARN [2016-10-07 15:52:32.02]: It's not recommended to set the `destination` value directly
WARN [2016-10-07 15:52:32.02]: Instead use the other options available in `scan --help`
WARN [2016-10-07 15:52:32.02]: Using your value 'platform=iOS Simulator,name=iPhone 5s' for now
WARN [2016-10-07 15:52:32.02]: because I trust you know what you're doing...
DEBUG [2016-10-07 15:52:32.02]: Detected derived data path '/Users/0xced/Library/Developer/Xcode/DerivedData/XCDYouTubeKit-bmurvnzoezsufrhkbydjpsjovanl'

+-----------------------+--------------------------------------------------------------------------+
|                                     Summary for scan 0.13.1                                      |
+-----------------------+--------------------------------------------------------------------------+
| project               | XCDYouTubeKit.xcodeproj                                                  |
| scheme                | XCDYouTubeKit Framework                                                  |
| derived_data_path     | /Users/0xced/Library/Developer/Xcode/DerivedData/XCDYouTubeKit-bmurvn... |
| clean                 | false                                                                    |
| code_coverage         | false                                                                    |
| address_sanitizer     | false                                                                    |
| skip_build            | false                                                                    |
| output_directory      | ./test_output                                                            |
| buildlog_path         | ~/Library/Logs/scan                                                      |
| open_report           | false                                                                    |
| skip_slack            | false                                                                    |
| slack_only_on_failure | false                                                                    |
| use_clang_report_name | false                                                                    |
+-----------------------+--------------------------------------------------------------------------+

INFO [2016-10-07 15:52:32.02]: $ set -o pipefail && env NSUnbufferedIO=YES xcodebuild -scheme XCDYouTubeKit\ Framework -project XCDYouTubeKit.xcodeproj -destination 'platform=iOS Simulator,name=iPhone 5s' -derivedDataPath '/Users/0xced/Library/Developer/Xcode/DerivedData/XCDYouTubeKit-bmurvnzoezsufrhkbydjpsjovanl' build test | tee '/Users/0xced/Library/Logs/scan/XCDYouTubeKit-XCDYouTubeKit Framework.log' | xcpretty 
INFO [2016-10-07 15:52:32.02]: ▸ Loading...
INFO [2016-10-07 15:52:33.13]: ▸ Building XCDYouTubeKit/XCDYouTubeKit Framework [Debug]
INFO [2016-10-07 15:52:33.13]: ▸ Check Dependencies
INFO [2016-10-07 15:52:33.81]: ▸ Processing Info.plist
INFO [2016-10-07 15:52:33.82]: ▸ Compiling XCDYouTubePlayerScript.m
INFO [2016-10-07 15:52:35.49]: ▸ Compiling XCDYouTubeClient.m
INFO [2016-10-07 15:52:35.49]: ▸ Compiling XCDYouTubeVideoOperation.m
INFO [2016-10-07 15:52:35.49]: ▸ Compiling XCDYouTubeVideo.m
INFO [2016-10-07 15:52:35.50]: ▸ Compiling XCDYouTubeVideoWebpage.m
INFO [2016-10-07 15:52:35.50]: ▸ Compiling XCDYouTubeLogger.m
INFO [2016-10-07 15:52:35.50]: ▸ Compiling XCDYouTubeVideoPlayerViewController.m
INFO [2016-10-07 15:52:36.44]: ▸ Linking XCDYouTubeKit
INFO [2016-10-07 15:52:36.54]: ▸ Generating 'XCDYouTubeKit.framework.dSYM'
INFO [2016-10-07 15:52:36.63]: ▸ Copying XCDYouTubeClient.h
INFO [2016-10-07 15:52:36.63]: ▸ Copying XCDYouTubeError.h
INFO [2016-10-07 15:52:36.63]: ▸ Copying XCDYouTubeKit.h
INFO [2016-10-07 15:52:36.63]: ▸ Copying XCDYouTubeLogger.h
INFO [2016-10-07 15:52:36.63]: ▸ Copying XCDYouTubeOperation.h
INFO [2016-10-07 15:52:36.63]: ▸ Copying XCDYouTubeVideo.h
INFO [2016-10-07 15:52:36.63]: ▸ Copying XCDYouTubeVideoOperation.h
INFO [2016-10-07 15:52:36.63]: ▸ Copying XCDYouTubeVideoPlayerViewController.h
INFO [2016-10-07 15:52:36.64]: ▸ Running script 'Analyzer'
INFO [2016-10-07 15:52:36.69]: Running Tests: ▸ Touching XCDYouTubeKit.framework
INFO [2016-10-07 15:52:36.70]: ▸ Build Succeeded
INFO [2016-10-07 15:52:36.76]: ▸ 2016-10-07 15:52:36.766 xcodebuild[95600:10536841] [MT] IDETestOperationsObserverDebug: (ADCDA27B-6611-45B8-A731-0235E074E397) Beginning test session ADCDA27B-6611-45B8-A731-0235E074E397 with Xcode 7D1012 on target <DVTiPhoneSimulator: 0x7f86484382d0> {
INFO [2016-10-07 15:52:36.76]: ▸ SimDevice: SimDevice : iPhone 5s (EFB29DA0-14ED-4C9D-9904-0001EA8A6A0F) : state={ Shutdown } deviceType={ SimDeviceType : com.apple.CoreSimulator.SimDeviceType.iPhone-5s } runtime={ SimRuntime : 9.3 (13E230) - com.apple.CoreSimulator.SimRuntime.iOS-9-3 }
INFO [2016-10-07 15:52:36.76]: ▸ } (9.3 (13E230))
INFO [2016-10-07 15:52:36.76]: ▸ 2016-10-07 15:52:36.767 xcodebuild[95600:10536848]  IDETestOperationsObserverDebug: Writing diagnostic log for test session to:
INFO [2016-10-07 15:52:36.76]: ▸ /Users/0xced/Library/Developer/Xcode/DerivedData/XCDYouTubeKit-bmurvnzoezsufrhkbydjpsjovanl/Logs/Test/0DFA1484-06B2-4967-8199-F3CD4CB347A1/Session-2016-10-07_15:52:36-HKQtF5.log
INFO [2016-10-07 15:52:36.80]: ▸ Building XCDYouTubeKit/XCDYouTubeKit Framework [Code Coverage]
INFO [2016-10-07 15:52:36.80]: ▸ Check Dependencies
INFO [2016-10-07 15:52:37.02]: ▸ Processing Info.plist
INFO [2016-10-07 15:52:37.04]: ▸ Compiling XCDYouTubeVideoOperation.m
INFO [2016-10-07 15:52:37.35]: ▸ Compiling XCDYouTubePlayerScript.m
INFO [2016-10-07 15:52:38.25]: ▸ Compiling XCDYouTubeClient.m
INFO [2016-10-07 15:52:38.42]: ▸ Compiling XCDYouTubeLogger.m
INFO [2016-10-07 15:52:38.42]: ▸ Compiling XCDYouTubeVideo.m
INFO [2016-10-07 15:52:38.43]: ▸ Compiling XCDYouTubeVideoPlayerViewController.m
INFO [2016-10-07 15:52:38.99]: ▸ Compiling XCDYouTubeVideoWebpage.m
INFO [2016-10-07 15:52:39.00]: ▸ Linking XCDYouTubeKit
INFO [2016-10-07 15:52:39.16]: ▸ Generating 'XCDYouTubeKit.framework.dSYM'
INFO [2016-10-07 15:52:39.27]: ▸ Copying XCDYouTubeClient.h
INFO [2016-10-07 15:52:39.29]: ▸ Copying XCDYouTubeError.h
INFO [2016-10-07 15:52:39.29]: ▸ Copying XCDYouTubeKit.h
INFO [2016-10-07 15:52:39.30]: ▸ Copying XCDYouTubeLogger.h
INFO [2016-10-07 15:52:39.31]: ▸ Copying XCDYouTubeOperation.h
INFO [2016-10-07 15:52:39.32]: ▸ Copying XCDYouTubeVideo.h
INFO [2016-10-07 15:52:39.32]: ▸ Copying XCDYouTubeVideoOperation.h
INFO [2016-10-07 15:52:39.34]: ▸ Copying XCDYouTubeVideoPlayerViewController.h
INFO [2016-10-07 15:52:39.34]: ▸ Running script 'Analyzer'
INFO [2016-10-07 15:52:39.43]: Running Tests: ▸ Touching XCDYouTubeKit.framework
INFO [2016-10-07 15:52:39.45]: ▸ Building XCDYouTubeKit/XCDYouTubeKit Static Library [Code Coverage]
INFO [2016-10-07 15:52:39.45]: ▸ Check Dependencies
INFO [2016-10-07 15:52:39.45]: ▸ Copying XCDYouTubeKit/XCDYouTubeVideo.h
INFO [2016-10-07 15:52:39.45]: ▸ Copying XCDYouTubeKit/XCDYouTubeOperation.h
INFO [2016-10-07 15:52:39.45]: ▸ Copying XCDYouTubeKit/XCDYouTubeLogger.h
INFO [2016-10-07 15:52:39.45]: ▸ Copying XCDYouTubeKit/XCDYouTubeKit.h
INFO [2016-10-07 15:52:39.45]: ▸ Copying XCDYouTubeKit/XCDYouTubeClient.h
INFO [2016-10-07 15:52:39.45]: ▸ Copying XCDYouTubeKit/XCDYouTubeError.h
INFO [2016-10-07 15:52:39.45]: ▸ Copying XCDYouTubeKit/XCDYouTubeVideoOperation.h
INFO [2016-10-07 15:52:39.45]: ▸ Copying XCDYouTubeKit/XCDYouTubeVideoPlayerViewController.h
INFO [2016-10-07 15:52:39.45]: ▸ Compiling XCDYouTubeClient.m
INFO [2016-10-07 15:52:39.45]: ▸ Compiling XCDYouTubeVideoOperation.m
INFO [2016-10-07 15:52:39.45]: ▸ Compiling XCDYouTubeVideo.m
INFO [2016-10-07 15:52:39.45]: ▸ Compiling XCDYouTubeLogger.m
INFO [2016-10-07 15:52:39.45]: ▸ Compiling XCDYouTubePlayerScript.m
INFO [2016-10-07 15:52:39.45]: ▸ Compiling XCDYouTubeVideoPlayerViewController.m
INFO [2016-10-07 15:52:39.46]: ▸ Compiling XCDYouTubeVideoWebpage.m
INFO [2016-10-07 15:52:39.46]: ▸ Building library libXCDYouTubeKit.a
INFO [2016-10-07 15:52:39.46]: ▸ Running script 'Analyzer'
INFO [2016-10-07 15:52:39.47]: ▸ Running script 'Export Environment for XcodeCoverage'
INFO [2016-10-07 15:52:39.48]: ▸ Building XCDYouTubeKit/XCDYouTubeKit Tests [Code Coverage]
INFO [2016-10-07 15:52:39.49]: ▸ Check Dependencies
INFO [2016-10-07 15:52:39.59]: ▸ Compiling XCTestWorkarounds.m
INFO [2016-10-07 15:52:39.90]: ▸ Compiling VCR+NSURLSessionConfiguration.m
INFO [2016-10-07 15:52:40.18]: ▸ Compiling VCR.m
INFO [2016-10-07 15:52:40.20]: ▸ Compiling VCRCassette.m
INFO [2016-10-07 15:52:40.20]: ▸ Compiling VCRCassetteManager.m
INFO [2016-10-07 15:52:40.20]: ▸ Compiling VCRErrorSerialization.m
INFO [2016-10-07 15:52:40.20]: ▸ Compiling VCROrderedMutableDictionary.m
INFO [2016-10-07 15:52:40.20]: ▸ Compiling VCRRecording.m
INFO [2016-10-07 15:52:40.51]: ▸ Compiling VCRRecordingURLProtocol.m
INFO [2016-10-07 15:52:40.52]: ▸ Compiling VCRReplayingURLProtocol.m
INFO [2016-10-07 15:52:40.56]: ▸ Compiling VCRRequestKey.m
INFO [2016-10-07 15:52:40.65]: ▸ Compiling XCDYouTubeProtectedVideosTestCase.m
INFO [2016-10-07 15:52:41.60]: ▸ Compiling XCDYouTubeClientTestCase.m
INFO [2016-10-07 15:52:41.60]: ▸ Compiling XCDYouTubeKitTestCase.m
INFO [2016-10-07 15:52:41.60]: ▸ Compiling XCDYouTubeVideoPlayerViewControllerTestCase.m
INFO [2016-10-07 15:52:41.60]: ▸ Compiling XCDYouTubeVideoOperationTestCase.m
INFO [2016-10-07 15:52:41.60]: ▸ Compiling XCDYouTubeVideoTestCase.m
INFO [2016-10-07 15:52:41.60]: ▸ Linking XCDYouTubeKit
INFO [2016-10-07 15:52:41.75]: ▸ Generating 'XCDYouTubeKit.xctest.dSYM'
INFO [2016-10-07 15:52:41.82]: ▸ Copying XCDYouTubeKit\ Tests/Cassettes
INFO [2016-10-07 15:52:42.02]: ▸ Running script 'Delete *.gcda files'
INFO [2016-10-07 15:52:42.04]: Running Tests: ▸ Touching XCDYouTubeKit.xctest
INFO [2016-10-07 15:52:48.45]: ▸ All tests
INFO [2016-10-07 15:52:48.45]: ▸ Test Suite XCDYouTubeKit.xctest started
INFO [2016-10-07 15:52:48.45]: ▸ XCDYouTubeClientTestCase
INFO [2016-10-07 15:52:48.95]: ▸ ✓ testCancelingOperation (0.488 seconds)
INFO [2016-10-07 15:52:49.20]: ▸ ✓ testConnectionError_offline (0.259 seconds)
INFO [2016-10-07 15:52:49.23]: ▸ ✓ testExpiredLiveVideo (0.023 seconds)
INFO [2016-10-07 15:52:49.23]: ▸ ✓ testFrenchClient (0.005 seconds)
INFO [2016-10-07 15:52:49.43]: ▸ ✓ testGeoblockedVideo (0.199 seconds)
INFO [2016-10-07 15:52:49.55]: ▸ ✓ testInvalidVideoIdentifier (0.118 seconds)
INFO [2016-10-07 15:52:49.55]: ▸ ✓ testLiveVideo (0.003 seconds)
INFO [2016-10-07 15:52:49.56]: ▸ ✓ testMobileRestrictedVideo (0.006 seconds)
INFO [2016-10-07 15:52:49.56]: ▸ ✓ testNilCompletionHandler (0.001 seconds)
INFO [2016-10-07 15:52:49.70]: ▸ ✓ testNilVideoIdentifier (0.139 seconds)
INFO [2016-10-07 15:52:49.71]: ▸ ✓ testNonExistentVideoIdentifier (0.005 seconds)
INFO [2016-10-07 15:52:49.71]: ▸ ✓ testRemovedVideo (0.005 seconds)
INFO [2016-10-07 15:52:50.07]: ▸ ✓ testRestrictedVideo (0.356 seconds)
INFO [2016-10-07 15:52:50.19]: ▸ ✓ testSpaceVideoIdentifier (0.121 seconds)
INFO [2016-10-07 15:52:50.20]: ▸ ✓ testThatVideoHasMetadata (0.006 seconds)
INFO [2016-10-07 15:52:50.20]: ▸ ✓ testThatVideoIsAvailalbeOnDetailPageEventLabel (0.007 seconds)
INFO [2016-10-07 15:52:50.21]: ▸ ✓ testUsingClientOnNonMainThread (0.005 seconds)
INFO [2016-10-07 15:52:50.21]: ▸ XCDYouTubeProtectedVideosTestCase
INFO [2016-10-07 15:52:50.35]: ▸ ✓ testAgeRestrictedUnratedVideo (0.140 seconds)
INFO [2016-10-07 15:52:50.49]: ▸ ✓ testAgeRestrictedVEVOVideo (0.143 seconds)
INFO [2016-10-07 15:52:50.50]: ▸ ✓ testAgeRestrictedVEVOVideoWithEmbedWebPageConnectionError_offline (0.007 seconds)
INFO [2016-10-07 15:52:50.50]: ▸ ✓ testAgeRestrictedVideo (0.005 seconds)
INFO [2016-10-07 15:52:50.68]: ▸ ✓ testDASHAudioWithRateBypassIsPlayable (0.177 seconds)
INFO [2016-10-07 15:52:50.86]: ▸ ✓ testProtectedVEVOIsPlayable (0.178 seconds)
INFO [2016-10-07 15:52:51.03]: ▸ ✓ testProtectedVEVOVideo1 (0.165 seconds)
INFO [2016-10-07 15:52:51.19]: ▸ ✓ testProtectedVEVOVideo2 (0.164 seconds)
INFO [2016-10-07 15:52:51.36]: ▸ ✓ testProtectedVEVOVideo3 (0.168 seconds)
INFO [2016-10-07 15:52:51.50]: ▸ ✓ testProtectedVideoWithBrokenSignatureFunction_offline (0.135 seconds)
INFO [2016-10-07 15:52:51.55]: ▸ ✓ testProtectedVideoWithNonAnonymousJavaScriptPlayerFunction_offline (0.052 seconds)
INFO [2016-10-07 15:52:51.57]: ▸ ✓ testProtectedVideoWithoutJavaScriptPlayerURL_offline (0.020 seconds)
INFO [2016-10-07 15:52:51.69]: ▸ ✓ testProtectedVideoWithoutSignatureFunction_offline (0.126 seconds)
INFO [2016-10-07 15:52:51.71]: ▸ ✓ testProtectedVideoWithPlayerScriptConnectionError_offline (0.012 seconds)
INFO [2016-10-07 15:52:51.71]: ▸ ✓ testProtectedVideoWithWebPageConnectionError_offline (0.003 seconds)
INFO [2016-10-07 15:52:51.71]: ▸ XCDYouTubeVideoOperationTestCase
INFO [2016-10-07 15:52:51.92]: ▸ ✓ testCancelingOperationAfterDelay (0.205 seconds)
INFO [2016-10-07 15:52:51.92]: ▸ ✓ testCancelingOperationAfterStart (0.001 seconds)
INFO [2016-10-07 15:52:52.12]: ▸ ✓ testCancelingOperationBeforeStart (0.204 seconds)
INFO [2016-10-07 15:52:52.32]: ▸ ✓ testCancelingOperationTwice (0.202 seconds)
INFO [2016-10-07 15:52:52.32]: ▸ ✓ testIsAsynchronous (0.001 seconds)
INFO [2016-10-07 15:52:52.32]: ▸ ✓ testIsConcurrent (0.001 seconds)
INFO [2016-10-07 15:52:52.47]: ▸ ✓ testStartingOnBackgroundThread (0.144 seconds)
INFO [2016-10-07 15:52:52.62]: ▸ ✓ testStartingOnMainThread (0.154 seconds)
INFO [2016-10-07 15:52:52.62]: ▸ ✓ testWrongInitializer (0.001 seconds)
INFO [2016-10-07 15:52:52.63]: ▸ XCDYouTubeVideoPlayerViewControllerTestCase
INFO [2016-10-07 15:52:52.65]: ▸ ✓ testAPIMisuseException (0.023 seconds)
INFO [2016-10-07 15:52:52.80]: ▸ ✓ testAsynchronousVideoNotification (0.151 seconds)
INFO [2016-10-07 15:52:52.86]: ▸ ✓ testNoStreamAvailableErrorNotification (0.057 seconds)
INFO [2016-10-07 15:52:52.87]: ▸ ✓ testPresentInView (0.015 seconds)
INFO [2016-10-07 15:52:53.26]: ▸ ✓ testRestrictedPlaybackErrorNotification (0.382 seconds)
INFO [2016-10-07 15:52:53.27]: ▸ ✓ testVideoNotification (0.017 seconds)
INFO [2016-10-07 15:52:53.27]: ▸ ✓ testWrongInitializer (0.001 seconds)
INFO [2016-10-07 15:52:53.27]: ▸ XCDYouTubeVideoTestCase
INFO [2016-10-07 15:52:53.28]: ▸ ✓ testDescription (0.005 seconds)
INFO [2016-10-07 15:52:53.28]: ▸ ✓ testEmptyInfo (0.000 seconds)
INFO [2016-10-07 15:52:53.28]: ▸ ✓ testIncompleteInfo (0.000 seconds)
INFO [2016-10-07 15:52:53.28]: ▸ ✓ testInvalidInfo (0.000 seconds)
INFO [2016-10-07 15:52:53.28]: ▸ ✓ testNilInfo (0.000 seconds)
INFO [2016-10-07 15:52:53.28]: ▸ ✓ testVideoAsKeyInDictionary (0.000 seconds)
INFO [2016-10-07 15:52:53.28]: ▸ ✓ testVideoEquality (0.001 seconds)
INFO [2016-10-07 15:52:53.28]: ▸   Executed 55 tests, with 0 failures (0 unexpected) in 4.806 (4.831) seconds
INFO [2016-10-07 15:52:53.29]: ▸ 
INFO [2016-10-07 15:52:53.38]: ▸ Test Succeeded
+--------------------+-------+
|        Test Results        |
+--------------------+-------+
| Number of tests    | 55    |
| Number of failures | 0     |
+--------------------+-------+
```

</details>

Does launching simulators only happen when run through fastlane or is there a bug in scan?
### Configuration Files

Scanfile:

``` ruby
project "XCDYouTubeKit.xcodeproj"
scheme "XCDYouTubeKit Framework"
```
### Environment
- Xcode 7.3.1
- scan 0.13.1

**Comment by KrauseFx** [comment]
Hey @0xced, thanks for the report. It seems like this is only being called when setting the `devices` value, instead of directly setting the `destination`:

```ruby
open_simulator_for_device(Scan.devices.first) if Scan.devices
```

Could you give that a try?

**Comment by KrauseFx** [comment]
I don't know the reasoning behind this, I'd have to look closer into this

**Comment by 0xced** [comment]
Unfortunately, using the `--device` argument doesn't work as expected. When I run `scan --device "iPhone 5s"` it ends up producing an xcodebuild command with `-destination 'platform=macOS'`.

I suspect scan might be confused by my [single scheme setup](http://promisekit.org/news/2016/08/Multiplatform-Single-Scheme-Xcode-Projects/).

**Comment by TKBurner** [comment]
@0xced Thanks for following up. Do you run into this issue in a single target app? Also, any interest in doing a PR for your first issue? Thanks!

**Comment by 0xced** [comment]
I already started working on it and broke a lot of tests. 😁 The issue is definitely about the single scheme setup. I hope I'll have a working solution soon.

**Comment by 0xced** [comment]
So, I progressed a bit but I needed to make a change to fastlane_core (I added an `os_type` property to the `Device` class). So now, when I run `rake` at the root of the project, a lot of tests do not pass because of this error:
```
undefined method `os_type' for #<FastlaneCore::DeviceManager::Device:0x007fca5cb14eb8>`
```

How can I run the tests with the local version of fastlane_core? Or should I first submit a PR for the change in fastlane_core, wait for a new release and then submit a second PR once I can update the dependency on the latest version of fastlane_core?

**Comment by KrauseFx** [comment]
@0xced to run it locally, you can make use of the `Gemfile`:

Currently it looks like this:
```ruby
source "https://rubygems.org"

gemspec

plugins_path = File.join(File.dirname(__FILE__), '..', 'fastlane', 'fastlane', 'Pluginfile')
eval(File.read(plugins_path), binding) if File.exist?(plugins_path)
```
To use your local version of `fastlane_core` update it to

```ruby
source "https://rubygems.org"

gemspec

plugins_path = File.join(File.dirname(__FILE__), '..', 'fastlane', 'fastlane', 'Pluginfile')
eval(File.read(plugins_path), binding) if File.exist?(plugins_path)

gem "fastlane_core", path: "../fastlane_core"
```
and run `bundle install` or `bundle update`

**Comment by 0xced** [comment]
But should I submit a PR with the Gemfile using the local fastlane_core? Because if I don’t, the tests will not pass.

**Comment by KrauseFx** [comment]
@0xced Oh, sorry I misunderstood. To make the tests pass, you need to submit multiple PRs, one per tool. We're gonna have to merge _fastlane_core_ first, then do a release, and only after that the build will pass 👍

**Comment by 0xced** [comment]
OK, so here’s the first PR: **[fastlane_core] Add an `os_type` property to the `Device`** class #6555.

**Comment by 0xced** [comment]
And here’s the second PR: **[scan] Handle single scheme Xcode projects** #6637

**Comment by TKBurner** [comment]
@0xced Thanks! We'll work with you there 👍 👍 Have a wonderful weekend!

**Comment by 0xced** [comment]
`FASTLANE_EXPLICIT_OPEN_SIMULATOR` is now working fine with single scheme setups and scan 0.14.1. 😀
